samples/cpp/benchmark_app/main.cpp

```diff
@@ -1261,8 +1261,10 @@ int main(int argc, char* argv[]) {
             statistics->dump();
 
         // Performance metrics report
-        slog::info << "Count:      " << iteration << " iterations" << slog::endl;
-        slog::info << "Duration:   " << double_to_string(totalDuration) << " ms" << slog::endl;
+        if (device_name.find("AUTO") != std::string::npos)
```

Comment:
Should we need to add warning that device switching will happen soon when retrieving the property ov::execution_devices at step 8?

Reply:
the logic behind is quite a threading thing, the exe devices may change or not change depending on the load status in background threads